### ISSUE TITLE
The project setup not working well, at least on Windows

### ISSUE THREAD

**Issue by merks** [eclipse-m2e/m2e-core#1347]
Either I end up with a sea or red like this:

![image](https://user-images.githubusercontent.com/208716/231969881-649785f0-e301-4402-a696-a750ac6b7b1c.png)

Or something smaller:

![image](https://user-images.githubusercontent.com/208716/231970073-0461665a-221e-41a8-8ad5-d2c0076e9091.png)

The TP looks odd too, with lots of duplicates of identical-looking things:

![image](https://user-images.githubusercontent.com/208716/231970327-7ef0b7f0-efe1-4ecf-9af8-afd408d56c5a.png)

I tried this testing with the latest 2023-06 M1 version of the IDE.  I just committed the catalog changes so they should be visible shortly...

**Comment by HannesWell** [comment]
> Either I end up with a sea or red like this:

Yes the resource generation for the maven-runtime component is unfortunately not very robust and especially after version version updates takes a few rounds of clean+rebuild.
What's often the quickest way to make it work again is to simply delete the the `m2e-maven-runtime` project and its two nested project from disk (don't delete the entire git repo!) and to re-run the Oomph setup for m2e, which re-imports the projects and regenerates the resources from scratch.

But I'll try to make it more robust in the future.


> Or something smaller:

Yes I noticed the two execution failures of the `rcptt-maven-plugin` too. @laeubi do you have any idea?

The third error is from your change for you initial state of https://github.com/eclipse-m2e/m2e-core/pull/1348 :)

> The TP looks odd too, with lots of duplicates of identical-looking things:

That's because multiple repos contain them. It's nothing we can do about in m2e. 

![grafik](https://user-images.githubusercontent.com/44067969/232209910-5a714df7-6e4e-40d6-bea2-bb7da8f0ba49.png)
